src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -737,7 +737,8 @@ private void ScrollViewer_ScrollChanged(object sender, ScrollChangedEventArgs e)
             {
                 var first = _scrollViewer.VerticalOffset;
                 var last = _scrollViewer.ViewportHeight + first;
-                if (_scrollViewer.ViewportHeight > 0 && last >= Items.Count)
+
+                if (_scrollViewer.ViewportHeight > 0 && last >= PackageItems.Count())
```

Comment:
Accessing PackageItems creates an array, and then Count() allocates. 

Do we have a cheaper way of doing this?